src/components/content-tab-import-export.tsx

```diff
@@ -86,20 +99,170 @@ export const ExportSite = ( {
 				</p>
 			</div>
 			<div className="gap-4 flex flex-row">
-				<Button onClick={ onExportFullSite } variant="primary">
+				<CoreButton onClick={ onExportFullSite } variant="primary">
```

Comment:
Changed ! https://github.com/Automattic/studio/pull/369/commits/877e6aa799f35c2039ad7bab79cb0e7139168fbf
I used CoreButton because that was the button introduced on https://github.com/Automattic/studio/pull/365/files#diff-fd133fa5ff9b8ef225c2775f90a5122225974fff6288e81b6fd518ef9bbb2e0bR1